internal/host/hosts_suite_test.go

```diff
@@ -11,6 +12,7 @@ import (
 func TestHost(t *testing.T) {
 	RegisterFailHandler(Fail)
 	common.InitializeDBTest()
+	time.Sleep(10 * time.Second)
```

Comment:
why we need sleep? we should not have it

Reply:
Oops! Left some debug stuff in...deleted!